atd-vzv/src/Components/Popover/InfoPopover.js

```diff
@@ -56,6 +55,22 @@ const InfoPopover = ({ config }) => {
         <FontAwesomeIcon className="info-icon" icon={faInfoCircle} />
       </StyledInfoIcon>
     </Tippy>
+  ) : (
+    <span>
+      <FontAwesomeIcon icon={faInfoCircle} onClick={toggle} />
+      <Modal
+        isOpen={modal}
+        toggle={toggle}
+        zIndex={1305} // Set z-index to supercede SideDrawer and SideMapControlDateRange components
```

Comment:
This value seems trivial, but as long as it works 👍

Reply:
Good catch, yep, this isn't my favorite thing either. It becomes important when a modal is displayed at the same time as the mobile side control bar in the map view. Without this, the modal falls behind side bar and is not visible when clicking an info icon unless the side bar is closed (the MUI side bar starts its z-index at 1300 for a reason that I assume is important to MUI).